### PR TITLE
reset highlighted zone on drag

### PR DESCRIPTION
**Pull request by lory87**
**This pull request fixes the following issues**:

|BEFORE|AFTER|
|---|---|
|screen shot 2016-01-12 at 15 10 13 2|screen shot 2016-01-13 at 09 57 26 2|

### CODE REVIEW

File: filer/private/sass/components/_drag-and-drop.scss [hunk 1]
```diff
@@ -334,3 +334,6 @@ form .form-row {
     text-overflow: ellipsis;
     white-space: nowrap;
 }
+&.reset-hover {
```
Comment (dhorelik) [inline filer/private/sass/components/_drag-and-drop.scss]: why the `&`? I don't think there is any nesting involved, no parent above in the file

Comment (lory87) [inline filer/private/sass/components/_drag-and-drop.scss]: thanks forgot to remove ;)

**Comment by dhorelik** [comment]
lgtm

**Comment by lory87** [comment]
There are some problems on travis. @mikek is already on it. gulp ci is ok so I merge this pr

File: filer/static/filer/js/addons/table-dropzone.js [hunk 2]
```diff
@@ -60,14 +60,16 @@
...
                         infoMessage.removeClass(hiddenClass);
-                        dropzone.addClass(dragHoverClass);
+                        dropzone.addClass(dragHoverClass).removeClass('reset-hover');
 
                         folderName.text(folderTitle);
                     },
-                    dragleave: function () {
+                    dragleave: function (e) {
+
+                        e.currentTarget.offsetWidth;
```
Comment (vxsx) [inline filer/static/filer/js/addons/table-dropzone.js]: this can be removed :)

**Pull request merged**